packages/ui/src/models/camel/kamelet-resource.test.ts

```diff
@@ -1,10 +1,10 @@
 import { kameletJson } from '../../stubs/kamelet-route';
 import { AddStepMode } from '../visualization/base-visual-entity';
 import { CamelComponentFilterService } from '../visualization/flows/support/camel-component-filter.service';
-import { createCamelResource } from './camel-resource';
 import { KameletResource } from './kamelet-resource';
 import { SourceSchemaType } from './source-schema-type';
-import { cloneDeep } from 'lodash';
+import { CamelKResourceFactory } from './camel-k-resource-factory';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
I think we need to keep the previous import, as we recently switched from `lodash/cloneDeep` to regular `lodash`

Reply:
aah, I missed that's different.